src/content/1.7/development/mail.md

```diff
@@ -44,3 +42,42 @@ class mymodulemycontrollerModuleFrontController extends ModuleFrontController
 {{% notice tip %}}
 Prestashop will use the Shop Configuration to decide if use `smtp` connection or php `mail` function so check it out on backoffice or in `app/config/parameter.php`
 {{% /notice %}}
+
+## Add custom template
+
+`Mail::send` has some parameters. You can specify your emails templates path of your module in parameter `templatePath`.
+
+In your module you must create the subfolder `mails` and then a sub folder with languages. Example: `modules\yourmodulename\mails\en` for english.
+
+In this folder you do create 2 files: one with extension `.html` and one with extension `.txt`.
+
+The name of the template files is in second parameter. In the under example the template name is `contact`. So you do create 2 files in mails subfolders of your module: `modules\yourmodulename\mails\en\contact.html` and `modules\yourmodulename\mails\en\contact.txt`.
+
+After installation, the templates email files are moved under the active folder theme: `theme\classic\modules\yourmodulename\mails\en\....`.
+
+```php
+class mymodulemycontrollerModuleFrontController extends ModuleFrontController
+{
+
+    public function initContent()
+    {
+        parent::initContent();
+        Mail::Send(
+            (int)(Configuration::get('PS_LANG_DEFAULT')), // defaut language id
+            'contact', // email template file to be use
+            ' Module Installation', // email subject
+            array(
+                '{email}' => Configuration::get('PS_SHOP_EMAIL'), // sender email address
+                '{message}' => 'Hello world' // email content
+            ),
+            Configuration::get('PS_SHOP_EMAIL'), // receiver email address
+            NULL, //receiver name
+            NULL, //from email address
+            NULL,  //from name
+            NULL, //file attachment
+            NULL, //mode smtp
+            '/modules/yourmodulename/mails' //custom template path
```

Comment:
Oh actually the `Mail::send` computes the module name from the template path and then rebuild the absolute path, so I guess it works. But maybe you should remove the starting `/` then to avoid any confusion?